### PR TITLE
Remove me

### PR DESCRIPTION
**Pull request by Orholt**

### CODE REVIEW

**Comment by michalwitek1232** [comment]
asd

**Comment by michalwitek1232** [comment]
Could you remove it? I push it by mistake

File: flutter/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -11,6 +11,9 @@
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
     <uses-permission android:name="android.permission.RECORD_AUDIO" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />
+    <uses-permission android:name="android.permission.READ_FRAME_BUFFER" />
+    <uses-permission android:name="android.permission.CAPTURE_VIDEO_OUTPUT" />
+    <uses-permission android:name="android.permission.CAPTURE_SECURE_VIDEO_OUTPUT" />
```
Comment (rustdesk) [inline flutter/android/app/src/main/AndroidManifest.xml:16]: Curiously，these permission is for what? Are you testing new screen capture method on Android?

Comment (Orholt) [reply flutter/android/app/src/main/AndroidManifest.xml:16]: Yes

Comment (rustdesk) [reply flutter/android/app/src/main/AndroidManifest.xml:16]: Please let us know if you have better solution.

Comment (rustdesk) [reply flutter/android/app/src/main/AndroidManifest.xml:16]: Can I know why do you wanna change?

Comment (Orholt) [reply flutter/android/app/src/main/AndroidManifest.xml:16]: I wanted to get rid of android screencast popup, but adding these permissions in basis works, with rustdesk signed with system key, i just want to test it before i do something on main repo

**Pull request closed without merging**